### ISSUE TITLE
Set up typef's and resolvers for Contact data

### ISSUE THREAD

**Issue by Sebastian2908-2007** [Sebastian2908-2007/my-resume#4]
***Queries***
* as a user/admin I should be able to see all contacts that have been submitted.
* as a user/admin I should be able to get a single contact.

***Mutations***
* as a user/admin I should be able to update any contact that is submitted.
* as a user/admin I should be able to delete any contact.
* as a user/admin I should be able to manually submit a contact.
* ***EDIT***(this will go in issue for video uploads resolvers and mutations) as a public user I should be able to submit my contact info as well as any relevant pictures or videos pertaining to my project. ***PROGRESS*** users can submit basic info ie. first name, last etc. this simultaneously creates an account.
* as a user I should be able to log out

***NOTE***
* At this point, I will probably set up an admin page(to make sure I can get my videos and pictures rendering on the client-side from my amazon s3 bucket) ***EDIT*** no I'm not doing this in this issue
*  as well as an admin login(using react-admin ***EDIT*** I am now doing full user functionality so to distinguish admin I just created a isAdmin field on my schema so react-admin package is now a no go) for a simple user ***ALSO*** there will be no simple user
* so basically I will be setting up S3 with Multer in this issue ***EDIT*** no multer instead node.js stream module instead... I think

***Something else to Consider***
* As I do more research I think I may need an npm package called graphql-upload 
* As I do more research I think I may need a node module called stream
* so I think I can add admin functionality by using a boolean field when creating an account ie. ***admin: Boolean*** on my contact schema.
* I should probably change the Contact model to User as well
* set ADMIN boolean in a mongoose pre.save function.
* use an environment variable containing the password that should indicate an admin.

**Comment by Sebastian2908-2007** [comment]
I'm closing this issue some things like fileUpload stuff will move to a separate issue ***Notes*** and ***considerations*** will be copied and put in ***My Project***
